maven-compat/src/main/java/org/apache/maven/artifact/resolver/DefaultArtifactResolver.java

```diff
@@ -105,18 +105,12 @@ public DefaultArtifactResolver()
         int threads = Integer.getInteger( "maven.artifact.threads", 5 );
         if ( threads <= 1 )
         {
-            executor = new Executor()
-            {
-                public void execute( Runnable command )
-                {
-                    command.run();
-                }
-            };
+            executor = Runnable::run;
         }
         else
         {
             executor = new ThreadPoolExecutor( threads, threads, 3, TimeUnit.SECONDS,
-                                               new LinkedBlockingQueue<Runnable>(), new DaemonThreadCreator() );
```

Comment:
This should be separate PR.

Reply:
why? sounds a bit nitpicking

Reply:
No need for a separate PR...

Reply:
Please decide whether to make a separate PR or not 😃

Reply:
Let's leave as-is. In the future, try to avoid unrelated changes.